flybywiresim/aircraft
# fix(A380x): Showcase + CPT & FO views fix

Fixes #9198

## Summary of Changes
Fixes the showcase camera view for num(13,16,17) & updates the CPT + FO cockpit view to be realistic.

## Screenshots
CPT Camera (Cockpit section Pilot menu): 

![image](https://github.com/user-attachments/assets/7bf160bf-28ff-4f3c-ba4e-3950dabdf5b3)

FO Camera (Cockpit section Pilot menu):

![image](https://github.com/user-attachments/assets/4a3bda1f-55be-4eaf-8d1b-7eead529367f)

Showcase (Showcase section Fixed menu) number 13

![image](https://github.com/user-attachments/assets/7154727d-bc8e-42c7-a4a8-0196ef089e83)

Showcase (Showcase section Fixed menu) number 16 

![image](https://github.com/user-attachments/assets/1bf5b44e-bcba-409a-b99e-65332d9f8521)

Showcase (Showcase section Fixed menu) number 17

![image](https://github.com/user-attachments/assets/098ec1e9-e43b-4631-ab40-ec82b23e72c4)

Discord username: LunakisLeaks

## Testing instructions

1. Spawn at the gate or at the runway.
2. Check that the views are the same as the [pictures above](##Screenshots).
3. (Showcase section Fixed menu) Confirm each view is in the correct number [[Pictures](##Screenshots)].
4. Make sure there are no glitches (ex. camera going through the window).


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
